vignettes/spelling.Rmd

```diff
@@ -102,7 +102,13 @@ Project based custom dictionaries are located within the project.
 This is required to make the project reproducible.
 What you can do is copy the `.dic` files from other projects.
 
-# Quarto projects
+## Rd files
+
+Rd files contain the information for the help files of functions in an R package.
+We recommend to use [Roxygen2](https://roxygen2.r-lib.org/) to write the documentation alongside the code of the functions.
+Building the package will generate the
```

Comment:
This sentence is not finished.

Reply:
done in [3143734](https://github.com/inbo/checklist/pull/94/commits/31437348c8d2c53c05ba7cb943b7dff81aa33055)